CRM/Event/Form/Participant.php

```diff
@@ -2139,8 +2138,8 @@ protected function sendReceipts($params, $total_amount, array $customFields, arr
     $sent = [];
     $notSent = [];
     $this->assign('module', 'Event Registration');
-    $this->assignEventDetailsToTpl($params['event_id'], CRM_Utils_Array::value('role_id', $params), CRM_Utils_Array::value('receipt_text', $params), $this->_isPaidEvent);
-
+    $this->assignEventDetailsToTpl($params['event_id'], CRM_Utils_Array::value('role_id', $params), CRM_Utils_Array::value('receipt_text', $params));
+    $this->assign('isPrimary', (int) $this->_isPaidEvent);
```

Comment:
Do we need this? I don't think it is used anywhere.

Reply:
Well it would still be present in people's templates if they haven't updated on upgrade - but I think I could comment to that effect

Reply:
Of course, good idea